### ISSUE TITLE
[CMake] Export target geometric_simu

### ISSUE THREAD

**Issue by jmirabel** [stack-of-tasks/dynamic_graph_bridge#70]
In #69 I removed the export of geometric_simu. First of all, let me remind that the exported target isn't used (and I don't see how it could be used). Now, if you want to reintroduce the export, here is a way of reproducing the issue.

Keeping the old installation path for *geometric_simu*, i.e. *prefix/bin*, use *roscore* in one terminal and *rosrun dynamic_graph_bridge geometric_simu* in another.

Using the current installation path for *geometric_simu*, i.e. *prefix/lib/dynamic_graph_bridge*, and exporting target *geometric_simu*, try to compile any package that depends on dynamic_graph_bridge.

**Comment by olivier-stasse** [comment]
I do not think this is an issue. If geometric_simu executable by another package this would create a run_depend entry in  the file package.xml of the other package and a ADD_PROJECT_DEPENDENCY in the CMakeLists.txt (or in the catkin_depends following the ROS way).
So I am closing the issue. Feel to repon it if you think I misunderstood it.

**Comment by nim65s** [comment]
I just need to understand how to do that, even if it's not useful right now :)
